mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/194 - zone security

Closes #194 

To test:
- Run any scenario and open Zone View
- Select a zone and make sure security rating is correct
- Change any security policy to see the security rating change
